ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -568,6 +577,7 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{app
                             });
 
                             const activeExtension = this.state.statusExtensionsMap[this.selectedExtension];
```

Comment:
might suggest renaming this to `activeStatusExt` for consistency and less confusion now that there are two types of extensions on this page